py-polars/polars/selectors.py

```diff
@@ -1371,7 +1375,7 @@ def first() -> SelectorType:
     └─────┴─────┴─────┘
 
     """
-    return _selector_proxy_(F.first(), name="first")
+    return _selector_proxy_(wrap_expr(plr.first()), name="first")
```

Comment:
I don't think it's worth using the Rust implementation directly here - we pay just a single `if not columns` check. But it's fine I guess.

Reply:
The pl.first/pl.last now redirect as `cs` first/last selectors (like exclude) so they can't refer back to themselves ;)